.github/workflows/lint-labs.yml

```diff
@@ -0,0 +1,42 @@
+name: Linting check (PennyLane Labs)
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+    paths:
+      - pennylane/labs/**
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  determine_runner:
+    if: github.event.pull_request.draft == false
+    name: Determine runner type to use
+    uses: ./.github/workflows/determine-workflow-runner.yml
+    with:
+      default_runner: ubuntu-latest
+
+  black-pylint:
+    if: github.event.pull_request.draft == false
+    needs: [determine_runner]
+    runs-on: ${{ needs.determine_runner.outputs.runner_group }}
+
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
+      - name: Install dependencies
+        run: pip install black pylint==2.7.4 isort==5.13.2
+
+      - uses: actions/checkout@v4
+
+      - name: Run Pylint (test files)
```

Comment:
We should probably also run black and isort.

Reply:
black and isort are already run as part of `format.yml` workflow! So no need to re-add them here

Reply:
But then wouldn't the `format.yml` workflow also run pylint?

Reply:
I've updated the root pylintrc file to ignore all of pennylane/labs/* , it should be within this PR

Reply:
Sorry for missing that. I'll resolve the conversation and approve 🎉